target/linux/ramips/dts/mt7621_yuncore_ax820.dts

```diff
@@ -0,0 +1,134 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "yuncore,ax820", "mediatek,mt7621-soc";
+	model = "YunCore AX820";
+
+	aliases {
+		led-boot = &led_status_green;
+		led-failsafe = &led_status_green;
+		led-running = &led_status_green;
+		led-upgrade = &led_status_green;
+		label-mac-device = &wifi0;
```

Comment:
This typically does not work without directly assigning the MAC address in DTS.

Use 02_network for label_mac instead.

Reply:
thanks, `label-mac-device` and `wifi0` label removed and label detection moved to `02_network`.